spec/model/post_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before :each do
+    @first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                              bio: 'Teacher from Mexico.', posts_counter: 0)
+    @first_user.save
+
+    @post = Post.create(author: @first_user, title: 'Hello', text: 'This is my first post', comments_counter: 0,
+                        likes_counter: 0)
+    @post.save
+  end
+  context 'Posts validations' do
+    it 'Title must not be blank.' do
+      @post.title = ''
+      expect(@post).to_not be_valid
+    end
+
+    it 'Title must not exceed 250 characters' do
+      @post.title = 'A' * 251
+      expect(@post).to_not be_valid
+    end
+
+    it 'CommentsCounter must be an integer greater than or equal to zero' do
+      @post.comments_counter = -1
+      expect(@post).to_not be_valid
+    end
+
+    it 'LikesCounter must be an integer greater than or equal to zero.' do
+      @post.likes_counter = -1
+      expect(@post).to_not be_valid
+    end
```

Comment:
- [ ] This test fails because you have not added the validation for the likes counter in your Post model. Kindly validation for it in the Post model.

  ```
  validates :likes_counter, numericality: { only_integer: true }
  validates :likes_counter, comparison: { greater_than_or_equal_to: 0 }
  ```
  
  ![lieks](https://user-images.githubusercontent.com/81468675/190830523-74329655-5b92-43ee-ac31-7407305005da.jpg)